### ISSUE TITLE
Attributes need to be unique

### ISSUE THREAD

**Issue by ermincelikovic** [IBM/carbon-components-svelte#134]
I'm running into an error when using anything that touches SkeletonText component:  
$ (plugin svelte) ParseError: Attributes need to be unique  

/src/components/SkeletonText/SkeletonText.svelte
```
...
  <p
    on:click
    on:mouseover
    on:mouseenter
    on:mouseleave
    class={cx('--skeleton__text', heading && '--skeleton__heading', className)}
    style={`width: ${width};`}
    {style} />
```  
  
There are basically two "style" props.

**Comment by metonym** [comment]
@ermincelikovic Thank you for raising this issue. Will investigate.

**Comment by ermincelikovic** [comment]
@metonym I would suspect that this became an issue since svelte 3.18.2  
See https://github.com/sveltejs/svelte/commit/507661325629ca2b79df9d0103e47d5bd316f8e2

**Comment by metonym** [comment]
@ermincelikovic Nice catch.

Fixed and released in [v0.3.2](https://www.npmjs.com/package/carbon-components-svelte)
